src/js/techreport/combobox.js

```diff
@@ -0,0 +1,261 @@
+class ComboBox {
+  constructor(element, data, selected) {
+    this.element = element;
+
+    this.data = data;
+    this.filteredData = data;
+    this.focusedOption = -1;
+    this.focusedOptionStr = '';
+    this.selected = selected || [];
+    this.maxSelected = 10;
+
+    this.updateContent();
+    this.monitorInput();
+
+    this.selected.forEach(name => this.showSelectedElement(name));
+    this.element.querySelector('input[type="text"][disabled]').removeAttribute('disabled');
+  }
+
+  updateContent(data) {
+    const rows = data || this.data;
+    // Add options in the dropdown list
+    const listbox = this.element.querySelector('[role="listbox"]');
+    rows.forEach((row, index) => {
+      const option = document.createElement('div');
+      option.setAttribute('role', 'option');
+      option.setAttribute('aria-selected', 'false');
+      option.dataset.name = row;
+      option.dataset.key = index;
+      option.textContent = row;
+      option.id = `${this.element.dataset.id}-${row}`;
+      const logo = document.createElement('img');
+      logo.setAttribute('alt', '');
+      logo.setAttribute('src', `https://cdn.httparchive.org/static/icons/${row}.png`);
+      logo.setAttribute('height', '14px');
```

Comment:
Height attributes are not in pixels:

```suggestion
      logo.setAttribute('height', '14');
```

This is causing the Lighthouse fail